MyHW/views.py

```diff
@@ -0,0 +1,22 @@
+from django.shortcuts import render
+
+def hw1901(request):
+    messages = {request.POST.get('message01', ''):len(request.POST.get('message01', '')),
+                request.POST.get('message02', ''):len(request.POST.get('message02', '')),
+                request.POST.get('message03', ''):len(request.POST.get('message03', ''))}
+    keymax = max(messages, key=messages.get)
+    return render(request, 'HW_1901.html', context={'result':keymax})
+
+def hw1902(request):
+    date = request.POST.get('date', '')
```

Comment:
please use datetime.strftime function to parse dates

Reply:
done